src/lib/expression/evaluation/like_matcher.hpp

```diff
@@ -1,10 +1,14 @@
 #pragma once
 
-#include <experimental/functional>
+#include <functional>
 #include <regex>
 #include <utility>
 #include <variant>
 
```

Comment:
Add a todo to re-check when we update GCC and Clang?
Do you know which compiler complains?

Reply:
iirc GCC-9 did not have the matcher in the `functional` header yet. But I can double-check and everything might be fine if we update GCC anyways (so we don't need the `exerimental` include/feature check at all then). However, it was clang-17 on macOS (so libc++ 17) that does not have the `experimental` header anymore.

Reply:
Nope, GCC9 works as well. Removed the whole thing.